superset-frontend/src/dashboard/components/nativeFilters/FilterBar/ActionButtons/index.tsx

```diff
@@ -85,6 +86,7 @@ const ActionButtonsContainer = styled.div`
 `;
 
 export const ActionButtons = ({
+  width,
```

Comment:
```suggestion
  width = OPEN_FILTER_BAR_WIDTH
```

Nit: let's assign a default value as soon as possible

Reply:
sounds good